waspc/tools/install_libs_to_data_dir.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+# Helper to compile the waspc/libs/* packages locally and in CI.
+# It will then move it into the Cabal data dir (and thus, the installer archive in CI releases).
+
+script_dir=$(CDPATH="" cd -- "$(dirname -- "$0")" && pwd)
+waspc_dir=$script_dir/..
+data_libs_dir=$waspc_dir/data/Generator/libs
```

Comment:
Why is this location one level deeper than the one we use for packages?

Also, these two scripts are awfully similar. One also contains some information that might apply to both of them. Is there a way we can generalize and use the same script? Or does the `tgz` story make that impractical?